draft-ietf-httpbis-digest-headers.md

```diff
@@ -337,25 +337,29 @@ The BNF for "parameter" is as is used in [RFC7230]. All digest-algorithm values
 are case-insensitive.
 
 The Internet Assigned Numbers Authority (IANA) acts as a registry for
-digest-algorithm values.  The registry contains the following tokens.
+The registry contains the tokens listed below.
 
+Some algorithms, although registered, have since been proven to be vulnerable:
+the MD5 algorithm MUST NOT be used as it has been found vulnerable to collision
+attacks [CMU-836068], the SHA algorithm is NOT RECOMMENDED as it has been found
+vulnerable to collision attacks [IACR-2019-459].
 
   SHA-256:
-  :  
+  :
     * Description: The SHA-256 algorithm [RFC6234].  The output of
       this algorithm is encoded using the base64 encoding [RFC4648].
     * Reference: [RFC6234], [RFC4648], this document.
     * Status: standard
 
   SHA-512:
-  :  
+  :
```

Comment:
The trailing whitespace is actually needed after all of these colons in order to make the list line up all nicely (I discovered this yesterday). I think that requiring trailing whitespace is a smell that something is broken but I don't know how to fix the root cause.

Reply:
Yep. The editor removes them every time and I've to `vim` to avoid that.